std/typecons.d

```diff
@@ -4399,7 +4399,19 @@ unittest
 }
 
 /**
-Make proxy for $(D a).
+    Creates a proxy for the value `a` that will forward all operations
+    while disabling implicit conversions. The aliased item `a` must be 
+    an $(B lvalue). This is useful for creating a new type from the 
```

Comment:
Once I fix https://issues.dlang.org/show_bug.cgi?id=14328, these should be turned into links.